tests/languages/sas/format.test

```diff
@@ -0,0 +1,35 @@
+put x $hex16.;
+format salary uscurrency.;
+format=dollar12.;
+----------------------------------------------------
+
+[
+  ["altformat",
+    [
+      ["keyword", "put"],
+      " x ",
+      ["format", "$hex16."],
+      ["punctuation", ";"]
+    ]

```

Comment:
Please fix the formatting of your tests. The JSON should also be indented using tabs.

Btw. You can have the JSON generated for you. I a test fails, it will show you the full token stream. 
With the command `npm run test:languages -- --pretty` this token stream will be formatted. You can then just copy and past the token stream and done. Well, almost done; you still have to adjust the indentation from spaces to tabs.